### ISSUE TITLE
Some widget titles are underlined while some are not and that needs to be consistent

### ISSUE THREAD

**Issue by TanyaM** [wso2/analytics-apim#1123]
**Description:**
Please refer the below screenshot where "Api Usage Summary" widget title is not underlined and "Top Api Creators" widget is underlined. This inconsistent behavior can be observed in other widgets too and that needs to be made consistent.

<img width="458" alt="widget_title" src="https://user-images.githubusercontent.com/3957104/80935780-d9417c80-8deb-11ea-8932-65df001376d1.png">

**Comment by TanyaM** [comment]
It was decided to remove the underlines in all widgets and make the widget title centered

**Comment by ruwiniwj** [comment]
@TanyaM  can you please add the names of the widgets you have already fixed this for?

**Comment by TanyaM** [comment]
Sample code snippet is as follows. 
```html
<div style={styles.headingWrapper}>
      <h3  style={styles.heading}>
          <FormattedMessage id='api.info.heading' defaultMessage='OVERALL API INFO' />
      </h3>
      <p style={styles.subheading}>
          <FormattedMessage id='api.info.subheading' defaultMessage='(Last 7 Days)' />
      </p>
</div>
```
Header and SubHeading styles are as follows.

```css
 heading: {
    margin: 'auto',
    textAlign: 'center',
    fontWeight: 'normal',
    letterSpacing: 1.5,
    paddingBottom: '10px',
    marginTop: 0,
 },
 subheading: {
    textAlign: 'center',
    margin: 5,
    fontSize: 14,
    color: '#b5b5b5',
 }
```

**Comment by TanyaM** [comment]
Currently fixed in APIMOverallApiInfo widget and APIMApiAvailability widget. Let's update the issue as we proceed.

**Comment by ruwiniwj** [comment]
This has been fixed in 3.2.0 M1 release
